src/ppx/grfx/vk/vk_device.cpp

```diff
@@ -256,6 +274,8 @@ Result Device::ConfigureFeatures(const grfx::DeviceCreateInfo* pCreateInfo, VkPh
     features.shaderStorageImageMultisample        = foundFeatures.shaderStorageImageMultisample;
     features.samplerAnisotropy                    = foundFeatures.samplerAnisotropy;
 
+    mHasMultiView = pCreateInfo->multiView && foundMultiViewFeatures.multiview;
```

Comment:
It feels a little strange that `pCreateInfo->multiview` is only used here, I think we should either:
1) Leave the field there and only enable extension and feature if pCreateInfo->multiview is true (so if it is requested)
or
2) Just not have a field in DeviceCreateInfo, and enable multiview if it is supported.